merlin/study/celeryadapter.py

```diff
@@ -407,9 +524,8 @@ def verify_args(spec, worker_args, worker_name, overlap, disable_logs=False):
         nhash = ""
         if overlap:
             nhash = time.strftime("%Y%m%d-%H%M%S")
-        # TODO: Once flux fixes their bug, change this back to %h
         # %h in Celery is short for hostname including domain name
-        worker_args += f" -n {worker_name}{nhash}.%%h"
+        worker_args += f" -n {worker_name}{nhash}.%h"
```

Comment:
Don’t change this until we’ve confirmed it works

Reply:
I tried it out with the flux_par example and it seemed to work. I can switch it back for now though.